reservations/models.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class Reservation(models.Model):
+    created_at = models.DateTimeField(verbose_name=_('created at'), auto_now_add=True, editable=False)
```

Comment:
`editable=False` is not needed when `auto_now_add=True`

Reply:
alright! thanks!